### ISSUE TITLE
Can we also get the smalltv-ultra apps/features?

### ISSUE THREAD

**Issue by ticsi** [GeekMagicClock/smalltv-pro#103]
Hi,

I have both the pro and the ultra version of this gadget, and I really miss some of the ultra's features from this one. Things like the "simple weather clock", the 3-day weather forecast app, plus the setting to auto switch apps periodically. Is it possible to bring those features to the pro version as well?
![Screenshot 2024-10-04 112507](https://github.com/user-attachments/assets/bdfcbaa8-41a2-4f89-ba51-66b4e3aecca5)

Many thanks!

**Comment by Toon-nooT** [comment]
+1

**Comment by captainbracket** [comment]
+1 awesome toys!

**Comment by mnimocks** [comment]
Yes, I would like to choose which apps to switch to automatically like the SmallTVUltra does.

**Comment by lionB777** [comment]
+1

**Comment by hackra76** [comment]
+1
